USER_MANUAL.md

```diff
@@ -153,7 +153,14 @@ The ReplicatedStartupActorTest is failing, pending https://improbableio.atlassia
 * Tests a fix for UNR-1259: Fix of the WorkerFlags data structure not being per worker. When running through Unreal Editor using single process, different worker types can read other worker type's flags. As a result flags of different worker types with the same name get the wrong value.
 * Demonstrates that ClientWorkers and UnrealWorker read the correct value for the "test" worker flag, when both types have a "test" flag with different value.
 * Validation:
-  1. Use workerflags_testgym_config launch config to run multiplayer through the editor, for every worker running, different values based on their worker type should get printed/logged.
+  1. Navigate to: Project Settings->SpatialOS GDK for Unreal->Editor Settings->Launch.
```

Comment:
make this list:
1.
1.
1.

It will render as 

1.
2.
3.

this makes editing easier.